python/mleap/sklearn/ensemble/forest.py

```diff
@@ -89,8 +89,8 @@ def serialize_to_bundle(self, transformer, path, model):
         attributes.append(('num_features', transformer.n_features_))
         attributes.append(('tree_weights', tree_weights))
         attributes.append(('trees', ["tree{}".format(x) for x in range(0, len(transformer.estimators_))]))
-        if transformer.n_outputs_ > 1:
-            attributes.append(('num_classes', transformer.n_outputs_)) # TODO: get number of classes from the transformer
+        if transformer.n_classes_ > 1:
```

Comment:
what was the failure you were seeing?

i'm asking because we have this example https://github.com/combust/mleap-demo/blob/master/notebooks/airbnb-price-regression-scikit.ipynb that includes a random forest as well, and i could score the serialized model with mleap-scoring.

Reply:
I see now that that the demo uses a RandomForestRegressor instead of a RandomForestClassifier.

Reply:
I was indeed for a RandomForestClassifier. The  regressor has no classes and thus no issues.

Reply:
okay, that makes sense. if it's ok with you, could you please update here the if statement as well to check the type of the transformer rather than the n_classes?

Reply:
sure. I'll do that.

Reply:
Sorry, I was a bit busy, but I have updated the check and pushed it.